src/DynamoCore/Extensions/ExtensionDefinition.cs

```diff
@@ -10,5 +10,7 @@ public class ExtensionDefinition
 
         [DataMember]
         public string TypeName { get; set; }
+
+        public bool RequiresSignedEntryPoint { get; internal set; } = false;
```

Comment:
needs a summary

Reply:
hmmmmm - does this effect existing extensions and their automatic serialization and deserialization?
Can you show a sample?

Reply:
doesn't this seem kind of fragile?...

Reply:
hmmm... Didn't realize we automatically deserialize.  But yes that would be not good

Reply:
We don't ever serialize or deserialize to this class automatically.  We had build this class object always.  But regardless, this does not have to be a public property.  I will move it and the ViewExtension version to internal

Reply:
hmm - I see - Wonder why the dataMember attributes were used.

So you don't intend for this property to be deserialized?